Windows.Dockerfile

```diff
@@ -83,17 +84,24 @@ RUN $url = ('https://golang.org/dl/go{0}.windows-amd64.zip' -f $env:GOLANG_VERSI
 	\
 	Write-Host 'Complete.';
 
-# Build the docker executable
-FROM builder as dockercli
-ARG DOCKER_CLI_LKG_COMMIT=c98c4080a323fb0e4fdf7429d8af4e2e946d09b5
-WORKDIR \\gopath\\src\\github.com\\docker\\cli
-RUN git clone https://github.com/docker/cli.git \gopath\src\github.com\docker\cli; \
-	git checkout $env:DOCKER_CLI_LKG_COMMIT; \
-	scripts\\make.ps1 -Binary -ForceBuildAll
+# Download the docker executable
+FROM base as dockercli
```

Comment:
@northtyphoon where is this `base` image stored?

Reply:
It's a multi-stage dockerfile. base is defined in the first stage 
```
ARG WINDOWS_IMAGE=mcr.microsoft.com/windows/servercore:1903
FROM $WINDOWS_IMAGE as base
```

---
In reply to: [344888115](https://github.com/Azure/acr-builder/pull/543#discussion_r344888115) [](ancestors = 344888115)

Reply:
Ahh I see, thanks!